assets/js/main.js

```diff
@@ -94,7 +94,7 @@ $(document).ready((function ($) {
             $input.addClass('is-invalid');
 
             if (fieldID === 'detail') {
-                $('.auth-form').find('input').each(function () {
+                $('.auth-form' || '.appointment-form').find('input').each(function () {
```

Comment:
Try not to go over 79 chars in a single line:
![mroomapp – main js 2022-01-12 23-06-38](https://user-images.githubusercontent.com/12160621/149221515-45eaae47-5fe7-4dfb-8544-4a81d3d5c589.png)

![mroomapp – main js 2022-01-12 23-07-27](https://user-images.githubusercontent.com/12160621/149221580-7c39322f-4d9a-4783-9ff7-d3296d4a49a2.png)

Reply:
Changed

Reply:
Resolved.